gulpfile.js

```diff
@@ -20,6 +20,7 @@ var distDir = './dist/';
 var appsDir = './apps/';
 var debugDir = './debug/';
 var releaseDir = './release/';
+var installerDir = './installer/';
```

Comment:
I think we should place this into the `release/` directory, instead of the Windows zip file - I don't see much value in keeping the zip file for Windows around if we have a proper installer.

Reply:
I'm not too sure. Do you want that `release`create the installer and *not* the zip? Maybe we can create both and let the user install the version that he wants.

Reply:
I am not sure if there is need for the zip - we are also not publishing a zip for macos. I think the 'release' folder should contain whatever it is we are publishing as our 'official' release, and for that purpose I like the installer better.
But as you say, we could publish both.

Reply:
I will remove the zip by the moment, the function that generates it remains (is used by linux for the moment), so we can add it again if we need to.